packages/mobile/src/qrcode/schema.ts

```diff
@@ -19,6 +19,7 @@ export const UriDataType = ioType({
   currencyCode: ioUnion([ioUndefined, ioKeyof(LocalCurrencyCode)]),
   amount: ioUnion([ioUndefined, ioString]),
   comment: ioUnion([ioUndefined, ioString]),
+  asset: ioUnion([ioUndefined, ioString]),
```

Comment:
nit: What about using the term `token` instead `asset`?

Reply:
I like it! Changed :)